hieradata/node/ruka04.ls.lsst.org.yaml

```diff
@@ -0,0 +1,2 @@
+---
+profile::core::kernel::version: "3.10.0-1127.8.2.el7.x86_64"
```

Comment:
In `hieradata/cluster/ruka.yaml` There's an explicit kernel version applied to the cluster. Since ruka04 is part of the cluster is this setting needed?

Reply:
not going to use that cluster definition, cause I'm going to use an Antu node.

Reply:
I'm going to use the antu cluster definition, and that one doesn't have the kernel patch you applied